### ISSUE TITLE
Maybe wrong grad clipping

### ISSUE THREAD

**Issue by gmftbyGMFTBY** [taesunwhang/UMS-ResSel#3]
https://github.com/taesunwhang/UMS-ResSel/blob/e261f5c7d31bcac92e9ef72a47e7a1727226fc91/post_train/post_training.py#L162

Hi, first of all, thank you for your wonderful work, I am very confused about the `gradient clipping` position.
To the best of my knowledge, the grad clipping should be placed before `optimizer.step`, or it will never be used.

But I am not sure about this issue. Can you explain this question to me?

**Comment by taesunwhang** [comment]
Hi, thank you for your interest in our work.
With respect to `gradient clipping`, as you pointed out, it should be placed before `optimizer.step`.
Thank you for pointing it out and I apologize for this confusion. I have fixed this https://github.com/taesunwhang/UMS-ResSel/commit/825214d0a83bb0f936ae089639454e7eaeeeddbf.

**Comment by gmftbyGMFTBY** [comment]
Thank you for your response!
